### ISSUE TITLE
Install issues with netcoredb

### ISSUE THREAD

**Issue by samhedin** [emacs-lsp/dap-mode#781]
Hi, 
trying to get netcoredb working and encountered issues.
First, running `dap-netcore--debugger-install` results in
```
LSP :: Finished downloading /tmp/netcoredbgis2hpo.tar.gz...

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
```
So I went to https://github.com/Samsung/netcoredbg/releases/tag/3.1.0-1031 and downloaded `netcoredbg-linux-amd64.tar.gz`. I extracted the  folder to `dap-netcore-install-dir`: `/home/sam/.emacs.d/.cache/lsp/netcoredbg`
There might be a naming issue here since the executable and folder are both named `netcoredbg". In the end, I nested it, but it's all here:
```
~/.emacs.d/.cache/lsp/netcoredbg$ tree
.
└── netcoredbg
    ├── libdbgshim.so
    ├── Microsoft.CodeAnalysis.CSharp.dll
    ├── Microsoft.CodeAnalysis.CSharp.Scripting.dll
    ├── Microsoft.CodeAnalysis.dll
    ├── Microsoft.CodeAnalysis.Scripting.dll
    ├── netcoredbg
    └── SymbolReader.dll
```
and it can be found by `(dap-netcore--debugger-cmd) => "~.emacs.d/.cache/lsp/netcoredbg/netcoredbg/netcoredbg"`
The same result is achieved with ` (dap-netcore--debugger-locate-or-install)`

However, running the debugger still says `Netcoredbg is not installed. Do you want to install it?` but then it crashes with `aborted (core dumped)`.

So in short, `dap-netcore--debugger-cmd` finds the executable just fine, but I'm still prompted to install the server, which doesn't work.

The name of the downloaded file when installing the server is strange, I don't know why it's named `netcoredbgis2hpo.tar.gz` instead of `netcoredbg-linux-amd64.tar.gz` but I also don't really understand how github releases work :shrug:

**Comment by samhedin** [comment]
The easiest change was to just download netcoredbg and place it on the path, so closing since that's good enough for me :)
